plugins/out_s3/s3.c

```diff
@@ -739,11 +736,6 @@ static int cb_s3_init(struct flb_output_instance *ins,
 
     tmp = flb_output_get_property("compression", ins);
     if (tmp) {
-        if (ctx->use_put_object == FLB_FALSE) {
```

Comment:
we need to add a check here that if `ctx->use_put_object == FLB_FALSE && compress_arrow = true`, then log error

Reply:
Got it. will do